maltoolbox/language/languagegraph.py

```diff
@@ -925,7 +920,8 @@ def _from_dict(cls, serialized_graph: dict) -> LanguageGraph:
                                 expr_chain_dict,
                                 lang_graph
                             )
-                            attack_step.own_requires.append(expr_chain)
+                            if expr_chain:
+                                attack_step.own_requires.append(expr_chain)
```

Comment:
Shouldn't this just send out an exception when it fails to generate the step expression instead?